app/main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+from typing import List
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        self.skills: List[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.learn_skill("CSS")
+        self.learn_skill("HTML")
+        self.learn_skill("JavaScript")
```

Comment:
Do not duplicate code three times, extend the skills list defined in the `SoftwareEngineer` class with one line using the list of values and a `+=` operator or `extend` method